plugins/processors/t128_filter/t128_filter.go

```diff
@@ -10,20 +14,36 @@ const sampleConfig = `
   ## The conditions that must be met to pass a metric through. This is similar
   ## behavior to a tagpass, but the multiple tags are ANDed
   [[processors.t128_filter.condition]]
+    ## Mode dictates how to match the condition's tag values
+	## Valid values are:
+	##  * "exact": exact string comparison
+	##  * "glob": go flavored glob comparison (see https://github.com/gobwas/glob)
+	##  * "regex": go flavored regex comparison
+    # mode = "exact"
 
   [processors.t128_filter.condition.tags]
-     #tag1 = ["value1", "value2"]
-	 #tag2 = ["value3"]
+     # tag1 = ["value1", "value2"]
+	 # tag2 = ["value3"]
 
   [[processors.t128_filter.condition]]
+	# mode = "exact"
 
   [processors.t128_filter.condition.tags]
-     #tag1 = ["value3"]
+     # tag1 = ["value3"]
 `
 
 type tags map[string][]string
+type mode string
+
+const (
+	EMPTY_MODE mode = ""
```

Comment:
1. These don't follow the typical go naming convention (camel case)
2. It's totally personal preference, but I like to use `modeEmpty`, `modeExact` etc. so autocomplete will show me all the available modes. Take it or leave it.

Reply:
Oh true, I was getting used to Mist's way of doing the enums in their oc term repo, I'll change